snapshots/output/react/src/LoaderInput.tsx

```diff
@@ -4,16 +4,21 @@
   /** Takes loading prop, input component as child */
   interface Props {
 //          ^^^^^ definition react-example 1.0.0 src/`LoaderInput.tsx`/Props#
+//          documentation ```ts\nProps\n```
+//          documentation Takes loading prop, input component as child
```

Comment:
Is repeating the `documentation` on multiple lines intentional? TJ's PR uses a separate character from the second line.

Reply:
The `documentation` part is repeated for separate documentation elements. The formatting of multiline docstrings is different, here newlines are rendered as an escaped `\n` character
![CleanShot 2022-04-28 at 18 02 33](https://user-images.githubusercontent.com/1408093/165795407-719fffe5-e66f-4567-aa7e-b467c21eb54e.png)